src/pages/[platform]/build-a-backend/add-aws-services/logging/set-up-logging/index.mdx

```diff
@@ -104,29 +104,20 @@ You will need to create a log group in Amazon CloudWatch to send logs to. You ca
 
 Below is a sample CDK construct to create the Amazon CloudWatch log group as well as creating and assigning the permission policies to Amplify roles.
 
-The `<log-group-name>` and `<region>` configured in the CDK construct will be used later to initialize the Amplify Logger plugin.
+Create the `RemoteLoggingConstraintsConstruct` custom CDK construct as the following:
 
-### Replace the placeholder values with your own values:
-
-- `<log-group-name>` is the log group that logs will be sent to. Note that this CDK construct sample includes logic to create the CloudWatch log group that you may have already created in previous steps.
-- `<amplify-authenticated-role-name>` and `<amplify-unauthenticated-role-name>` are Amplify roles created as part of Amplify Auth configuration via Amplify CLI.
-
-```ts
+```ts title="amplify/custom/RemoteLoggingConstraintsConstruct/resource.ts"
```

Comment:
can we name this just `RemoteLoggingConstraints`? the cdk pattern does not include the `Construct` suffix